packages/search/tests/integration/components/navi-search-result/definition-test.js

```diff
@@ -92,4 +92,49 @@ module('Integration | Component | definition', function(hooks) {
       'definition descriptions are shown correctly'
     );
   });
+
+  test('Multiple datasources', async function(assert) {
+    assert.expect(4);
+
+    const data = [
+      {
+        id: 'pageViews',
+        name: 'Page Views',
+        description: 'The number of views of a page.',
+        source: 'dummy'
+      },
+      {
+        id: 'impressions',
+        name: 'Impressions',
+        description: 'Number of times a user saw the ad.',
+        source: 'blockhead'
+      }
+    ];
+    const result = {
+      component: 'navi-search-result/definition',
+      title: 'Definitions',
+      data
+    };
+    set(this, 'result', result);
+
+    await render(hbs`<NaviSearchResult::Definition @data={{this.result.data}} />`);
+
+    assert.dom('.navi-search-definition-result').exists({ count: 2 }, 'Two result is displayed');
+
+    const definitionTitles = findAll('.navi-search-definition-result__item-name').map(el => el.textContent.trim());
+    const definitionDescriptions = findAll('.navi-search-definition-result__item-description').map(el =>
+      el.textContent.trim()
+    );
+    const definitionSources = findAll('.navi-search-definition-result__item-source').map(el => el.textContent.trim());
+    const expectedDefinitionTitles = ['Page Views', 'Impressions'];
+    const expectedDefinitionDescriptions = ['The number of views of a page.', 'Number of times a user saw the ad.'];
+    const expectedDefinitionSources = ['Source: dummy', 'Source: blockhead'];
+    assert.deepEqual(definitionTitles, expectedDefinitionTitles, 'definition titles are shown correctly');
```

Comment:
Kind of a nitpick, but can we make these like

```js
assert.deepEqual(
  findAll('.navi-search-definition-result__item-name').map(el => el.textContent.trim()),
  ['The number of views of a page.', 'Number of times a user saw the ad.'],
  'definition descriptions are shown correctly'
)
```

It's easier to read when the results/expected are right next to each other